package.json

```diff
@@ -17,7 +17,7 @@
     "build": "node node_modules/buildman/index.js --all",
     "travis-test": "istanbul cover ./tests/index.js && (coveralls < coverage/lcov.info || exit 0)",
     "cover-climate": "istanbul cover ./tests/index.js && (codeclimate < coverage/lcov.info || exit 0)",
-    "test": "faucet; jshint --verbose index.js"
+    "test": "node_modules/.bin/faucet; node_modules/.bin/jshint --verbose index.js"
```

Comment:
npm scripts will automatically add ./node_modules/.bin to the PATH https://docs.npmjs.com/misc/scripts#path